packages/wdio-runner/src/utils.js

```diff
@@ -73,10 +73,27 @@ export function sanitizeCaps (caps) {
     }, {})
 }
 
+/**
+ * checks whether capabilities contains multiple browser configurations
+ * @param  {Object}  capabilities  desired session capabilities
+ * @return {Boolean}               true if multiple browsers configured
+ */
+export function hasMultiremoteOptions (capabilities) {
```

Comment:
How about calling this `isMultiremoteSession`?

Reply:
how about `isMultiremoteCapability` then?

Reply:
I renamed it to `isMultiremoteConfig`, considering having capabilities both as argument and function name declaration is somewhat repetitive.